README.md

```diff
@@ -95,3 +95,86 @@ To ensure that NX is running the latest versions of all packages within the mono
+   - **Dry Run**: Set this to `true` for a dry run, which will simulate the release process without publishing.
+
+3. **Publishing Process**  
+   The workflow will:
+   - Run format, lint, test, and build checks on the specified package.
+   - Automatically version and publish the package to NPM if the `dry_run` is set to `false`.
+
+4. **Verify the Release**  
+   Once the workflow completes, confirm the package has been published on the NPM registry.
+
+## Contributing Code
+
+We welcome contributions from other teams to the **govuk-one-login-frontend** monorepo! Follow these guidelines to ensure consistency and quality across the repository.
+
+### Steps to Contribute
+
+1. **Clone The Repo**  
+
+   Clone the repository locally and navigate into the project directory:
+   ```bash
+   git clone git@github.com:govuk-one-login/govuk-one-login-frontend.git ./your_folder_name
+
+  Replace your_folder_name with your desired directory name. If you omit this part, it will clone into a folder named govuk-one-login-frontend.
+
+2. **Creating A Branch**
+
+After cloning, create a new branch with a descriptive name that reflects the changes you’re making. For example:
+
+ `git checkout -b ticket-number/your-feature-name`
+
+3. **Install Dependencies**
+
+Ensure all dependencies are installed, by running:
+
+  `npm install`
+
+4. **Make Your Changes**
+
+Add your code changes, ensuring that they align with GOV.UK OneLogin coding standards and best practices.
+
+5. **Run Tests and Linting**
+
+Before pushing your changes, run all tests and linting to make sure your code adheres to the monorepo's requirements:
+
+- If you only want to run tests in the affected package, `cd` into the package folder and run `npm test` or just `npm test` at the root level to run all test suites.
+- If there are any uncommitted changes, you will need to run `npx nx run-many -t test` as the tests and linters cache the results, so they will only run for changed cases.
+
+- [Lint the code with eslint+prettier:](#lint-the-code-with-eslintprettier)
+
+6. **Commit and Push Changes**
+
+Use **Conventional Commits** for your commit messages to ensure automatic versioning and changelog updates. This allows NX to automatically generate a changelog with merged PRs for each release, helping users track the latest changes when they update their packages.
+
+For example:
+
+- `feat(ticket-number): add new component for user authentication`
+- `fix(ticket-number): correct header alignment issue`
+- `docs(ticket-number): update README with contribution guidelines`
+
+7. **Open a Pull Request**
+
+We use a PR template to ensure all necessary details are captured. Simply fill in the details and check the relevant checkboxes. 
+
+8. **Review and Approval**
+
+Your PR needs to be reviewed by a member of the code owners, the digital-identity-frontend-capability team.Once approved can be merged into main.
```

Comment:
We could add a useful links section

https://gds-way.digital.cabinet-office.gov.uk/ 
https://design-system.service.gov.uk/ (for any new components)

https://govukverify.atlassian.net/wiki/spaces/DIFC/pages/3779199166/Coding+Standards (tbc if this is appropriate as its an open source project?)